ui/app/src/views/auth/SignWrapper.tsx

```diff
@@ -11,33 +11,72 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Divider, Stack } from '@mui/material';
+import { Button, Divider, Stack } from '@mui/material';
 import { ReactNode } from 'react';
 import { useDarkMode } from '../../context/DarkMode';
 import PersesLogoCropped from '../../components/logo/PersesLogoCropped';
 import DarkThemePersesLogo from '../../components/logo/DarkThemePersesLogo';
 import LightThemePersesLogo from '../../components/logo/LightThemePersesLogo';
 import { useIsLaptopSize } from '../../utils/browser-size';
+import { useConfigContext } from '../../context/Config';
 
 export function SignWrapper(props: { children: ReactNode }) {
   const { isDarkModeEnabled } = useDarkMode();
   const isLaptopSize = useIsLaptopSize();
+  const config = useConfigContext();
+  const oauthProviders = (config.config?.security?.authentication?.providers?.oauth_providers || []).map(
+    (provider) => ({ slug_id: `oauth-${provider.slug_id}`, name: provider.name })
+  );
+  const oidcProviders = (config.config?.security?.authentication?.providers?.oidc_providers || []).map((provider) => ({
+    slug_id: `oidc-${provider.slug_id}`,
+    name: provider.name,
+  }));
   return (
-    <Stack
-      width="100%"
-      flexDirection={isLaptopSize ? 'row' : 'column'}
-      alignItems="center"
-      justifyContent="center"
-      gap={2}
-    >
-      {!isLaptopSize ? <PersesLogoCropped /> : isDarkModeEnabled ? <DarkThemePersesLogo /> : <LightThemePersesLogo />}
-      <Divider
-        orientation={isLaptopSize ? 'vertical' : 'horizontal'}
-        variant="middle"
-        flexItem
-        sx={{ marginTop: isLaptopSize ? '30vh' : undefined, marginBottom: isLaptopSize ? '30vh' : undefined }}
-      />
-      <Stack gap={1}>{props.children}</Stack>
+    <Stack width="100%" alignItems="center" justifyContent="center" flexDirection={'row'}>
+      <Stack alignItems="center" justifyContent="center" flexDirection={'column'} gap={2}>
+        <Stack
+          width="100%"
+          flexDirection={isLaptopSize ? 'row' : 'column'}
+          alignItems="center"
+          justifyContent="center"
+          gap={2}
+        >
+          {!isLaptopSize ? (
+            <PersesLogoCropped />
+          ) : isDarkModeEnabled ? (
+            <DarkThemePersesLogo />
+          ) : (
+            <LightThemePersesLogo />
+          )}
+          <Divider orientation={isLaptopSize ? 'vertical' : 'horizontal'} variant="middle" flexItem />
+          <Stack gap={1}>{props.children}</Stack>
+        </Stack>
+        <Divider orientation={'horizontal'} variant="fullWidth" flexItem>
```

Comment:
I think it would be nice if this part is kept on the right side of the logo like the native auth.

It would avoid a weird moment when the native auth is not activated and you have nothing on the right.

Reply:
by the way you need to consider when the native auth is not enabled here.

Reply:
on a different part, would be cool to display the logo of each provider. Maybe we could add an attribute that would give the type of the provider. And we could support a list of provider logo (like GitHub, GitLab, Microsoft, Google, AWS as a starting base). WDYT ?

Reply:
Yes I didn't focus yet on the login page. I need to make sure it adapt to every situation (native+external, only native, only external) + also check the responsive which I guess is not working.

I agree that we should have logo indeed. We need to have all the known logos in the assets and somehow knows which one to use based on the config. I would wish to do that without adding yet another config line parsing urls for example but I'm not sure. Let me check what others do.

Reply:
I updated the sign in with a much simpler aproach. As you suggested all the sign in together at the right side. Code is simpler thanks to that. 

Regarding the logo. I propose you to add in a subsequent PR.